src/branding-bar/BrandingBar.js

```diff
@@ -20,6 +20,7 @@ const ICON_SIZE = "26px";
 const Y_MARGIN = "12px";
 
 const brandingBarFixedStyles = `
+  background: ${THEME.colors.background};
```

Comment:
FYI, kind of a weird situation: This `brandingBarFixedStyles` is ultimately used within a `mediaQuery` and seems to lose its ability to access `props`.  This seems to hold true even if styled-component `css` is used instead of standard string interpolation